EndlessClient/Dialogs/GuildDialog.cs

```diff
@@ -136,7 +150,100 @@ private void SetState(GuildDialogState state)
+                        {
+                            ShowIconBackGround = false,
+                            IconGraphic = _dialogIconService.IconSheet,
+                            IconGraphicSource = _dialogIconService.GetDialogIconSource(DialogIcon.GuildRanking),
+                            PrimaryText = _localizedStringFinder.GetString(EOResourceID.GUILD_RANKING),
+                            SubText = _localizedStringFinder.GetString(EOResourceID.GUILD_ASSIGN_RANK_TO_MEMBER),
+                            OffsetY = 45,
+                        };
+                        assignRankItem.LeftClick += (_, _) => CheckAndChangeState(GuildDialogState.AssignRank);
+                        assignRankItem.RightClick += (_, _) => CheckAndChangeState(GuildDialogState.AssignRank);
+
+                        AddItemToList(assignRankItem, sortList: false);
+
+                        var removeMemberItem = new ListDialogItem(this, ListDialogItem.ListItemStyle.Large, 0)
+                        {
+                            ShowIconBackGround = false,
+                            IconGraphic = _dialogIconService.IconSheet,
+                            IconGraphicSource = _dialogIconService.GetDialogIconSource(DialogIcon.GuildRemoveMember),
+                            PrimaryText = _localizedStringFinder.GetString(EOResourceID.GUILD_REMOVE_MEMBER),
+                            SubText = _localizedStringFinder.GetString(EOResourceID.GUILD_REMOVE_A_MEMBER_FROM_GUILD),
+                            OffsetY = 45,
+                        };
+                        removeMemberItem.LeftClick += (_, _) => CheckAndChangeState(GuildDialogState.RemoveMember);
+                        removeMemberItem.RightClick += (_, _) => CheckAndChangeState(GuildDialogState.RemoveMember);
+
+                        AddItemToList(removeMemberItem, sortList: false);
+
+                        var disbandItem = new ListDialogItem(this, ListDialogItem.ListItemStyle.Large, 0)
+                        {
+                            ShowIconBackGround = false,
+                            IconGraphic = _dialogIconService.IconSheet,
+                            IconGraphicSource = _dialogIconService.GetDialogIconSource(DialogIcon.GuildDisband),
+                            PrimaryText = _localizedStringFinder.GetString(EOResourceID.GUILD_DISBAND),
+                            SubText = _localizedStringFinder.GetString(EOResourceID.GUILD_DISBAND_YOUR_GUILD),
+                            OffsetY = 45,
+                        };
+                        disbandItem.LeftClick += (_, _) => CheckAndChangeState(GuildDialogState.Disband);
+                        disbandItem.RightClick += (_, _) => CheckAndChangeState(GuildDialogState.Disband);
+
+                        AddItemToList(disbandItem, sortList: false);
+                    }
+                    break;
+
             }
+
+        }
+
+        private void CheckAndChangeState(GuildDialogState state)
```

Comment:
Rename to something clearer: e.g. `SetStateIfGuildMember` (feel free to choose a better name if you think of one)